cunumeric/runtime.py

```diff
@@ -57,6 +68,8 @@
     np.complex128: ty.complex128,
 }
 
+ThunkArray = Union[DeferredArray, EagerArray]
```

Comment:
Not sure yet if this approach is required. Was having issues trying to use `NumpyThunk`

Reply:
my gut feeling is that this shouldn't be necessary

Reply:
https://github.com/nv-legate/cunumeric/pull/428/commits/53a02a9aa05eb9320e52274f0c01902d0ebb52be